python/mscclpp/comm.py

```diff
@@ -21,6 +21,7 @@
 )
 import mpi4py
 import numpy as np
+import torch
```

Comment:
Hmm... then our packages will depend on torch. Could we provide some utils such as `ptr_from_torch` to get the data_ptr?

Reply:
I agree, perhaps we can do something like this and use `TorchTensor` instead of `torch.Tensor`:
```python
try:
    import torch
    TorchTensor = torch.Tensor
except:
    class TorchTensor:
        pass
```